src/main/kotlin/ar/edu/unahur/obj2/semillasAlViento/Parcela.kt

```diff
@@ -18,14 +20,15 @@ class Parcela(val ancho: Int, val largo: Int, val horasSolPorDia: Int) {
       cantidadPlantas += 1
     }
   }
+  fun tieneComplicaciones() = plantas.any { it.horasDeSolQueTolera() < horasSolPorDia }
 }
 
 class Agricultora(val parcelas: MutableList<Parcela>) {
```

Comment:
Las Parcelas no cambiaban
```suggestion
class Agricultora(val parcelas: List<Parcela>) {
```